setuptools/msvc.py

```diff
@@ -1038,9 +1038,13 @@ def VCTools(self):
             tools += [os.path.join(si.VCInstallDir, path)]
 
         elif self.vs_ver >= 15.0:
-            host_dir = (
-                r'bin\HostX86%s' if self.pi.current_is_x86() else r'bin\HostX64%s'
-            )
+            if self.pi.current_is_x86():
+                host_dir = r'bin\HostX86%s'
+            elif self.pi.current_cpu == 'arm64':
```

Comment:
Note that this comparison is no longer symmetrical with the check for 'x86'. Better would be to be consistent. Either use `self.pi.current_cpu == 'x86'` inline or implement a new `self.pi.current_is_arm64()` method.

Reply:
Addressed.